code/modules/research/xenobiology/crossbreeding/destabilized.dm

```diff
@@ -0,0 +1,320 @@
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		L.Stun(70, TRUE, TRUE)
+		to_chat(L, "<span class='warning'>You suddenly feel every molecule in your body grind to a halt!</span>")
+	else
+		new /obj/effect/timestop(get_turf(A), null, 50)
+
+/obj/item/slimecross/destabilized/cerulean
+	colour = "cerulean"
+
+/obj/item/slimecross/destabilized/cerulean/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(ismovableatom(A))
+		new /mob/living/simple_animal/hostile/clone(get_turf(A), A, thrower)
+	else
+		new /mob/living/simple_animal/hostile/clone(get_turf(A), thrower, thrower)
+		
+/obj/item/slimecross/destabilized/pyrite
+	colour = "pyrite"
+
+/obj/item/slimecross/destabilized/pyrite/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	var/turf/T = get_turf(A)
+	T.add_atom_colour("#[rand_hex_color()]", WASHABLE_COLOUR_PRIORITY)
+	for(var/atom/atoms in T)
+		atoms.add_atom_colour("#[rand_hex_color()]", WASHABLE_COLOUR_PRIORITY)
+
+/obj/item/slimecross/destabilized/red
+	colour = "red"
+
+/obj/item/slimecross/destabilized/red/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		L.apply_status_effect(/datum/status_effect/heavyBleeding)
+	else
+		new /obj/effect/decal/cleanable/blood(get_turf(A))
+
+/obj/item/slimecross/destabilized/green
+	colour = "green"
+
+/obj/item/slimecross/destabilized/green/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A) && isliving(thrower))
+		var/mob/living/LTarget = A
+		var/mob/living/LThrower = thrower
+		var/damageToDeal
+		if(iscarbon(LThrower))
+			var/mob/living/carbon/CThrower = LThrower
+			var/damageType = findBiggestLoss(CThrower)
+			damageToDeal = min(CThrower.get_damage_amount(damageType), 25)
+			CThrower.apply_damage_type(-25, damageType)
+			if(iscarbon(LTarget))
+				var/mob/living/carbon/CTarget = LTarget
+				CTarget.apply_damage_type(damageToDeal, damageType)
+			else
+				LTarget.adjustBruteLoss(damageToDeal)
+		else
+			damageToDeal = -(LThrower.adjustBruteLoss(-25))
+			LTarget.adjustBruteLoss(damageToDeal)
+
+/obj/item/slimecross/destabilized/green/proc/findBiggestLoss(mob/living/carbon/C)
+	var/damageType = BRUTE
+	var/maxDamage = C.get_damage_amount(BRUTE)
+	var/burnDamage = C.get_damage_amount(BURN)
+	if(burnDamage > maxDamage)
+		maxDamage = burnDamage
+		damageType = BURN
+	var/toxinDamage = C.get_damage_amount(TOX)
+	if(toxinDamage > maxDamage)
+		maxDamage = toxinDamage
+		damageType = TOX
+	var/oxyDamage = C.get_damage_amount(OXY)
+	if(oxyDamage > maxDamage)
+		maxDamage = oxyDamage
+		damageType = OXY
+	var/cloneDamage = C.get_damage_amount(CLONE)
+	if(cloneDamage > maxDamage)
+		maxDamage = cloneDamage
+		damageType = CLONE
+	var/staminaDamage = C.get_damage_amount(STAMINA)
+	if(staminaDamage > maxDamage)
+		maxDamage = staminaDamage
+		damageType = STAMINA
+	var/brainDamage = C.get_damage_amount(BRAIN)
+	if(brainDamage > maxDamage)
+		maxDamage = brainDamage
+		damageType = BRAIN
+	return damageType
```

Comment:
what the fug :----DDDDDD

Reply:
@Tlaltecuhtli  Did I miss something or is this line of code particularly amusing to you for some reason, lol